docs/_articles/en/apex/writing.md

```diff
@@ -32,49 +43,55 @@ You can preview, view, or go to definitions of:
+When you refresh SObject definitions, the extension uses the default org to generate Apex classes. These classes represent the standard and custom objects that the current user has access to. We recommend that you don’t edit these representative Apex classes because these classes are intended to be read-only. If they’re modified, it may impact the auto-completion suggestions. Also, the changes are lost when you refresh the SObject definitions.
+
+Whenever you refresh SObject definitions, the representative Apex classes are deleted and regenerated. You can modify the SObjects either by updating the objects’ `.object-meta.xml` and `.field-meta.xml` files; Or by making changes declaratively in the default org. After modifying the SObjects, make sure to sync your local project and the default org.
 
-To preview a definition, hold down Ctrl (Windows or Linux) or Cmd (macOS) and hover over the item whose definition you want to see.
+When you launch the Salesforce CLI Integration extension (which is part of the Salesforce Extension Pack) for the first time and the `salesforcedx-vscode-apex.enable-sobject-refresh-on-startup` setting is enabled, `SFDX: Refresh SObject Definitions` command is executed automatically if your project doesn’t contain any Apex classes. After you add or edit standard or custom objects or their fields, make sure to run this command.
 
-To view a definition, right-click the item and select **Peek Definition**, or press Alt+F12.
+## Go To Definitions
 
-To jump to the location of a definition, right-click the item and select **Go to Definition**, or press F12.  
-![Previewing, viewing, and jumping to a definition](./images/apex_go_to_definition.gif)
+Apex extension provides go to definition support for user-defined Apex such as classes and methods, standard objects, and custom objects.
+
+- To preview a definition, press and hold Ctrl (Windows or Linux) or Cmd (macOS) and hover over the item whose definition you want to see.
+- To view a definition, right-click the item and select **Peek Definition**, or press Alt+F12.
+- To jump to the location of a definition, right-click the item and select **Go to Definition**, or press F12.
+
+![Previewing, viewing, and jumping to a definition](../../images/apex_go_to_definition.gif)
 
 ## Find All References
 
-You can find all references to user-defined Apex:
+You can find all references to user-defined Apex such as Classes, Class variables, Enums, Interfaces, Methods, and Properties. To find references, select an item and press Shift+F12; Or right-click the item and select **Find All References**. The reference results are displayed in the left pane of the editor window.
+
+## Outline View
+
+The Apex outline view shows the structure of the Apex class or trigger that’s open in the editor.
+
+- To view the list of the symbols in the file, press Ctrl+Shift+O (Windows or Linux) or Cmd+Shift+O (macOS).
```

Comment:
I know this shortcut was listed previously, but is this right? Cmd+Shift+0 doesn't bring anything up for me in an Apex class. I have not used this shortcut before, so just checking. @brpowell do you happen to know?

Reply:
I believe that's an 'O' not '0' :), the former works.

Reply:
@brpowell perfect - Thanks :)